### DISCUSSION TITLE
How can I implement for some set of routers rather than using it for all the routes?

### DISCUSSION THREAD

**Discussion by geeky01adarsh** [animir/node-rate-limiter-flexible#259] [Q&A] [answered]
I have a backend that serves many routes for different purposes, and not every request from there requires a rate limiter. I want to implement a `rate-limiter` for the `api` routes. 
Do you know how I can do that?
Also, will it support serverless deployment?

**Comment by animir** [comment]
@geeky01adarsh HI, You can find an example here on Wiki https://github.com/animir/node-rate-limiter-flexible/wiki/Overall-example#different-limits-for-different-parts-of-application
It should work in serverless environment.

**Reply by geeky01adarsh** [reply]
I have gone through that, but can you please guide me through how can I implement in on e on my router class, where I don't have to the express initialized app?

**Reply by animir** [reply, marked as answer]
You should create two or more limiters and use them on your router class depending on route.

**Reply by geeky01adarsh** [reply]
Ok got it!
Thanks a lot! I will try doing that, and will close the discussion after that.
